src/state/reducer.ts

```diff
@@ -13,18 +17,49 @@ import transactions from './transactions/reducer'
 import user from './user/reducer'
 import wallets from './wallets/reducer'
 
-export default {
-  application,
+const persistedReducers = {
   user,
   transactions,
   signatures,
+  lists,
+}
+
+const appReducer = combineReducers({
+  ...persistedReducers,
+  application,
   wallets,
   mint,
   mintV3,
   burn,
   burnV3,
   multicall: multicall.reducer,
-  lists,
   logs,
   [routingApi.reducerPath]: routingApi.reducer,
+})
+
+export type AppState = ReturnType<typeof appReducer>
+
+function createStorage() {
+  const db = localForage.createInstance({
+    name: 'redux',
```

Comment:
Can this be more specific, in case some other web app we publish on the same domain (uniswap.org) ends up also using redux?

```suggestion
    name: 'redux-interface',
```

Reply:
Alternatively (and possibly cleaner), continue to use `redux` as the db name, but change the `key` from `'root'` to `'interface'`.